packages/itwinui-css/src/tabs/classes.scss

```diff
@@ -13,6 +12,14 @@
   &.iui-vertical {
     @include iui-tabs-vertical;
   }
+
+  .iui-tabs-actions {
```

Comment:
I _think_ you should be able to un-nest `.iui-tabs-actions` & `.iui-tabs-content` to reduce specificity.

```scss
.iui-tabs-wrapper {
  …

  &.iui-horizontal { … }
  &.iui-vertical { … }
}

.iui-tabs-action { … }
.iui-tabs-content { … }
```

Reply:
Ohh, thanks, yes, I think it works. Because no other component would have `.iui-tabs-action` because of the "`iui-tabs`" prefix and also all tests passed.